src/test/java/org/junitpioneer/jupiter/StdIoExtensionTests.java

```diff
@@ -122,10 +153,55 @@ void catchesBoth(StdIn in, StdOut out) throws IOException {
 
 	}
 
-	@ExtendWith(StdIOExtension.class)
-	static class StdIOExtensionConfigurations {
+	@Nested
+	@TestMethodOrder(OrderAnnotation.class)
```

Comment:
Just a note:

This nested class should be thread safe due the use of `@TestMethodOrder` regarding to [JUnit 5 User Guide](https://junit.org/junit5/docs/current/user-guide/#writing-tests-parallel-execution)

> The default execution mode is applied to all nodes of the test tree with a few notable exceptions, namely test classes that use the Lifecycle.PER_CLASS mode or a MethodOrderer (except for Random).